test/check-machines-disks

```diff
@@ -835,6 +835,16 @@ class TestMachinesDisks(VirtualMachinesCase):
             xfail=True, xfail_error_title="Disk failed to be attached",
         ).execute()
 
+        # Using directory path for disk should detect disk type
+        m.execute("qemu-img create -f qcow2 /tmp/foobar.qcow2 1M")
+        self.VMAddDiskDialog(
+            self,
+            device='disk',
+            expected_target='vdb',
```

Comment:
Two blocks below there is a test case which adds a .qcow2 file already. (1) How does this work right now? and (2) how is qcow vs. raw related to the "vdb" vs. "sdb" target? bus type and image format should be completely independent from each other?

Can we also have a test case for raw format?

Reply:
Ok, that novell.qcow2 test needs fixing then, as it just touches a qcow2 file so that's wrong:

```
[jelle@t14s][~/projects/cockpit-machines]%touch /tmp/foo.qcow2
[jelle@t14s][~/projects/cockpit-machines]%qemu-img info /tmp/foo.qcow2
image: /tmp/foo.qcow2
file format: raw
virtual size: 0 B (0 bytes)
disk size: 0 B
```

Reply:
About the bus type, not sure. The expected_target needs to be passed as the test asserts that in `verify_disk_added`.

Reply:
Right, please fix that test below as well (separate commit), it's rather confusing. I suggest to just rename the file from `novell.qcow2` to `empty.img`.